newdemojail.yaml

```
@@ -0,0 +1,772 @@
+        column: "SubjectIdentification"
+      nc.PersonSurName:
+        type: "nc.PersonSurName"
+        column: "LastName"
+      nc.PersonGivenName:
+        type: "nc.PersonGivenName"
+        column: "FirstName"
+      nc.PersonMiddleName:
+        type: "nc.PersonMiddleName"
+        column: "MiddleName"
+      nc.PersonBirthDate:
+        type: "nc.PersonBirthDate"
+        column: "BirthDate"
+      nc.PersonSex:
+        type: "nc.PersonSex"
+        column: "Sex"
+      nc.SSN:
+        type: "nc.SSN"
+        column: "SocialSecurityNumber"
+      nc.PersonRace:
+        type: "nc.PersonRace"
+        column: "Race"
+      nc.PersonEthnicity:
+        type: "nc.PersonEthnicity"
+        column: "Ethnicity"
+    name: "inmate"
+  persondetails:
+    fqn: "ol.persondetails"
+    entitySetName: "DemoPersondetails"
+    propertyDefinitions:
+      general.id:
+        type: "general.id"
+        column: "SubjectIdentification"
+      person.maritalstatus:
+        type: "person.maritalstatus"
+        column: "MaritalStatus"
+      person.gender:
+        type: "person.gender"
+        column: "Gender"
+      person.juvenile:
+        type: "person.juvenile"
+        column: "Juvenile"
+    name: "persondetails"
+  education:
+    fqn: "ol.education"
+    entitySetName: "DemoEducation"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "SubjectIdentification"
+      # ol.highesteducation_enum:
+      #   type: "ol.highesteducation_enum"
+      #   column: "Education"
+    name: "education"
+  facility:
+    fqn: "ol.location"
+    entitySetName: "DemoFacility"
+    propertyDefinitions:
+      general.id:
+        type: "general.id"
+        column: "Facility"
+      location.name:
+        type: "location.name"
+        column: "Facility"
+    name: "facility"
+  jailstay:
+    fqn: "ol.jailstay"
+    entitySetName: "DemoJailstay"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "BookId"
+      incident.startdatetime:
+        type: "incident.startdatetime"
+        column: "SentenceStartDate"
+        transforms:
+        - !<transforms.DateAsDateTimeTransform>
+          pattern: ['yyyy-MM-dd']
+          timezone: "America/Los Angeles"
+      ol.projectedreleasedate:
+        type: "ol.projectedreleasedatetime"
+        column: "SentenceEndDate"
+        transforms:
+        - !<transforms.DateAsDateTimeTransform>
+          pattern: ['yyyy-MM-dd']
+          timezone: "America/Los Angeles"
+      ol.releasedate:
+        type: "ol.releasedatetime"
+        column: "ReleaseDate"
+        transforms:
+        - !<transforms.DateAsDateTimeTransform>
+          pattern: ['yyyy-MM-dd']
+          timezone: "America/Los Angeles"
+      criminaljustice.releasetype:
+        type: "criminaljustice.releasetype"
+        column: "ReleaseType"
+    name: "jailstay"
+  jailstaylength:
+    fqn: "ol.jailstaylength"
+    entitySetName: "DemoJailstaylength"
```

```suggestion
    entitySetName: "DemoTimeServed"
```

We prefer "time served" as more intuitive than "jailstay length". This is also in the wiki. But please check for an existing one - I haven't yet.

---

It exists and I've changed the flight to this entity set now 👍